### ISSUE TITLE
Possible bug in Higgs mass calculation

### ISSUE THREAD

**Issue by JoseEliel** [wilson-eft/wilson#57]
In the function m2Lambda_to_vMh2 (defined in line 38 of smpar.py) the Higgs mass formula  

```
    Mh2 = 2 * m2 * (1 - m2 / Lambda * (3 * C['phi'] - 4 * Lambda * C['phiBox'] +
                                        Lambda * C['phiD']))
```

seems to have a typo. Comparing with the expression from https://arxiv.org/abs/1704.03888, there should be a Lambda^2 term in the factor multiplying the second parenthesis, i.e. it should be 

```
Mh2 = 2 * m2 * (1 - m2 / (Lambda*Lambda) * (3 * C['phi'] - 4 * Lambda * C['phiBox'] + 
                                       Lambda * C['phiD']))

```

**Comment by DavidMStraub** [comment]
Oh. I don't see why this would *not* be a bug, indeed.

Does changing that solve #52?

**Comment by JoseEliel** [comment]
No, I have just checked. I think they are unrelated issues.

**Comment by DavidMStraub** [comment]
Waiting for @peterstangl or @jasonaebischerGIT to confirm.

Then it would be great if you can submit a PR (but please use `**2`, this is not Fortan :wink:)

**Comment by JoseEliel** [comment]
I will. Good point on `**2`, I'm often writing C++ (luckily not Fortran 😉 ), so that's where that came from.

**Comment by jasonaebischerGIT** [comment]
> ... there should be a Lambda^2 term in the factor multiplying the second parenthesis, i.e. it should be
> 
> ```
> Mh2 = 2 * m2 * (1 - m2 / (Lambda*Lambda) * (3 * C['phi'] - 4 * Lambda * C['phiBox'] + 
>                                        Lambda * C['phiD']))

I agree, this formula is correct.

**Comment by JoseEliel** [comment]
I've made the pull request.

**Comment by jasonaebischerGIT** [comment]
The check from travis seems to have failed. Not sure what the origin of this is...

**Comment by celis** [comment]
from the travis logs it seems one needs to reduce the numerical accuracy (places argument) required in the assertAlmostEqual method.

**Comment by peterstangl** [comment]
This issue should be solved by the new PR #59 and I think PR #58 can be closed.

BTW: Fortran actually *does* implement the `**` exponentiation operator. So this seems to be just an issue with C/C++ :wink: 
(that's why one can get relatively nice expressions that work in Python from Mathematica's `FortranForm`)

**Comment by jasonaebischerGIT** [comment]
Closing this issue, which is resolved by PR #59.
